### PR TITLE
plugins:digitalocean - Release 0.2.0 (DEB)

### PR DESCRIPTION
**Pull request by TJM**
This should be released to develop and 1.8 branches only.

### CODE REVIEW

File: plugins/ruby-foreman-digitalocean/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-digitalocean (0.2.0-1) stable; urgency=low
```
Comment (mmoll) [inline plugins/ruby-foreman-digitalocean/debian/changelog]: nitpick: add an empty line there

File: plugins/ruby-foreman-digitalocean/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+  * Version 0.2.0
+  * #10242 - Changed template name to _base for Foreman 1.8
+
+ -- Tommy McNeely <tommy@lark-it.com>  Fri, 13 Feb 2015 07:57:17 +0000
+
 ruby-foreman-digitalocean (0.1.0-1) stable; urgency=low
```
Comment (mmoll) [inline plugins/ruby-foreman-digitalocean/debian/changelog]: and there

**Comment by mmoll** [comment]
:+1:

**Comment by domcleal** [comment]
Merged as 960d989, thanks @TJM!

(By the way, there's a 'dch' program if you have Debian tools available, or scripts/changelog.rb in this repo can both add new changelog entries.)

**Pull request closed without merging**